tests/integration_tests/cpu/material/material_interaction.cpp

```diff
@@ -107,15 +112,11 @@ GTEST_TEST(detray_material, telescope_geometry_energy_loss) {
     ASSERT_TRUE(p.propagate(state, actor_states))
         << state.debug_stream.str() << std::endl;
 
-    // muon
-    const int pdg{interactor_state.pdg};
+    // new momentum
+    const scalar newP{ptc.charge() / state._stepping._bound_params.qop()};
```

Comment:
Is there a reason, this can't be:
```suggestion
    const scalar newP{state._stepping._bound_params.p(ptc.charge())};
```

Reply:
ok